Formula/wine.rb

```diff
@@ -38,8 +38,11 @@ def require_universal_deps?
     MacOS.prefer_64_bit?
   end
 
+  option "without-win64", "Build without 64-bit support"
+
   # Wine will build both the Mac and the X11 driver by default, and you can switch
   # between them. But if you really want to build without X11, you can.
+  depends_on :xcode => ["8.0", :build] if build.with? "win64"
```

Comment:
Last tweak: the comment above refers to `:x11` below so can you move this new line below `depends_on "pkg-config" => :build`. Post a comment when you've done that and CI is 💚 and I'll make sure to merge ASAP. Thanks!